app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="your-tasks">
+  <h2>YOUR TASKS</h2>
+  <ol>
+    <% @tasks.each do |task| %>
+      <li>
+        <% if task.complete == true %>
```

Comment:
The `== true` is redundant here because `if` statements are automatically checking for [truthiness](https://learn.co/lessons/truthiness-in-ruby-readme), and booleans such as `task.complete` has "truthiness" that matches its value of `true`/`false`.

Same for line 1 in `_action_bar.html.erb`.